backend/src/ds/group_operation.rs

```diff
@@ -391,6 +391,8 @@ impl DsGroupState {
                         .map_err(|_| GroupOperationError::LibraryError)?,
                 ),
                 client_reference: client_queue_config,
+                // No notifications for welcome messages
+                suppress_notifications: true.into(),
```

Comment:
Why do we want to suppress it for Welcome messages? We currently show a useful notification when we are added to a group.